tests/Main.hs

```diff
@@ -116,11 +116,12 @@ instance Show RelPath where
 instance Arbitrary RelPath where
   arbitrary = do
     p <-
-      intercalate "/"
-        <$> listOf1
-          ( (++) <$> vectorOf 3 charGen
-              <*> listOf1 charGen
-          )
+      resize 10 $
```

Comment:
Why `resize 10`?

Reply:
Currently, even w/o patch tests are failing on macOS on `createDirectoryIfMissing` because the generated path is too large. So there should be a way to reduce its size. Using resize was the simplest way to do that, another approach would be to shrink the generated path to the maximum allowed size. I have no strong preference here, and ready to change the code.